wsmaster/che-core-api-workspace/src/main/resources/schema/devfile.json

```diff
@@ -174,8 +192,21 @@
                     "description": "Describes the component id. It has the following format: [{REGISTRY_URL}/]{plugin/editor PUBLISHER}/{plugin/editor NAME}/{plugin/editor VERSION}, where {REGISTRY_URL}/ is an optional part.",
                     "pattern": "^((https?://)[a-zA-Z0-9_\\-./]+/)?[a-z0-9_\\-.]+/[a-z0-9_\\-.]+/[a-z0-9_\\-.]+$",
                     "examples": [
-                      "eclipse/maven-jdk8/1.0.0",
-                      "https://che-plugin-registry.openshift.io/eclipse/maven-jdk8/1.0.0"
+                      "eclipse/maven-jdk8/1.0.0"
+                    ]
+                  },
+                  "reference": {
+                    "description": "Describes raw location of plugin yaml file.",
+                    "type": "string",
+                    "examples": [
+                      "https://pastebin.com/raw/kYprWiNB"
+                    ]
+                  },
+                  "registryUrl": {
+                    "description": "Describes URL of custom plugin registry.",
```

Comment:
Do you think it makes sense to add `pattern` here?

Reply:
Added

Reply:
yep